tests/main/document-interfaces-url/task.yaml

```diff
@@ -38,15 +38,17 @@ execute: |
     # The expiry date is <expiration year>/<expiration month>      
     
     declare -A exclusion_map
-    exclusion_map["intel-qat"]="2024/10"
-    exclusion_map["microceph"]="2024/10"
-    exclusion_map["microceph-support"]="2024/10"
-    exclusion_map["pkcs11"]="2024/10"
-    exclusion_map["registry"]="2024/11"
```

Comment:
registry still needs and exception

Reply:
Huh, strange that the test didn't fail though. Maybe it got a 500 for an unrelated reason and just didn't test that url. Anyway, I've fixed this now, thank you